vm/context.ts

```diff
@@ -414,13 +414,9 @@ module J2ME {
       release || assert (Frame.isMarker(marker));
     }
 
-    executeFrames(group: Frame []) {
+    executeFrame(frame: Frame) {
```

Comment:
Nit: update the comment in vm/context.ts with the new function name (otherwise someone reading the comment could end up not finding the function)